pkg/apis/workspaces/v1alpha2/WorkspacePodContribution.go

```diff
@@ -4,7 +4,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-type WorkspacePodContributions struct {
+type DevWorkspacePodContributions struct {
```

Comment:
I think this rename isn't strictly necessary, but if it's a hassle to undo then I'm fine with it. Is this struct used by anyone?

Reply:
It's only DWO who uses it.
I can return it back but personally I would like a bit more to use DevWorkspace to be synced with CR kind.

Reply:
BTW File name should be synced with type )

Reply:
Well, I checked and it's unused. DWO has it's own impl https://github.com/devfile/devworkspace-operator/blob/main/apis/controller/v1alpha1/common.go#L18

If it's unused, let's remove it then?

Reply:
It seems like a strange thing to have in the spec API so I'm :+1: to remove as long as odo isn't using it.